src/common/settings.js

```diff
@@ -68,6 +68,13 @@ const options_list = {
         }
     },
 
+    "hide-results-on-whisper": {
+        "title": "Hide roll results on D&D Beyond",
+        "description": "Don't show the roll results on D&D Beyond when using whisper",
+        "type": "bool",
+        "default": false
+    },
+
```

Comment:
I disagree with this option because those who aren't using a VTT, will simply not see anything. This should be conditional on sending to Discord only, and the option's text should reflect that too.